src/devices/Arduino/ArduinoAnalogController.cs

```diff
@@ -35,60 +35,58 @@ public override int PinCount
             get;
         }
 
-        public override int ConvertPinNumberToLogicalNumberingScheme(int pinNumber)
+        /// <inheritdoc />
```

Comment:
I'm failing to understand how all of the changes in the Arduino device are related to this PR. Are those just there because when working on the sample you made some adjustments? Does it make sense to separate these changes into a separate PR?

Reply:
Yes, most of these changes are improvements for the example. I'm considering to split that into a separate PR, but I'm waiting until the design has been roughly finalized, to avoid diverging branches.

Reply:
Got it, yeah, we'd probably want those in a separate PR. If anything, it'll make reviewing this PR easier 😃